WordPress/Classes/Utility/Migrations/BlogToBlogSettings40to41.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+
+
+/**
+ *  @class      BlogToBlogSettings40to41
+ *  @brief      This class should be executed during the migration from Data Model Mark 40 >> 41.
+ *              It's main goal is to set the relationship between the *Blog* and *BlogSetting* instances,
+ *              in the destination context.
+ */
+class BlogToBlogSettings40to41: NSEntityMigrationPolicy
```

Comment:
Verified! NSEntityMigrationPolicy is gone for good. *THANKS* for so much awesomeness!!